### PR TITLE
WIP [DO NOT MERGE] [DEVELOPER-4710] Install Beryllium theme

### PR DESCRIPTION
**Pull request by rogeralloymagnetic**
**NOTE: This needs acceptance testing from content managers.**

### JIRA Issue Link
Spike: https://issues.jboss.org/browse/DEVELOPER-4697
Install: https://issues.jboss.org/browse/DEVELOPER-4710
Fix Views UI: https://issues.jboss.org/browse/DEVELOPER-5504

### Verification Process
- Login to admin and view new admin theme

screen shot 2018-11-30 at 1 01 05 pm

### CODE REVIEW

**Comment by LightGuard** [comment]
Seems okay so far, minus this error in the console: TypeError: $this.select2 is not a function

**Comment by LightGuard** [comment]
@rogeralloymagnetic Any ideas on that JS error I mentioned above?

File: _docker/drupal/drupal-filesystem/web/config/sync/beryllium.settings.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  url: ''
+  use_default: true
+table_responsive: '-1'
+tooltip_enabled: 0
+_core:
+  default_config_hash: a42sGCtaJGEVIf1d2gYT8a5hupmdlFuBsGnrP5F3gUE
+favicon:
+  mimetype: image/vnd.microsoft.icon
+  path: null
+  url: 'http://localhost:8888/themes/contrib/bootstrap/favicon.ico'
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply _docker/drupal/drupal-filesystem/web/config/sync/beryllium.settings.yml]: Good catch, Rob.

[the remark this answers is not in the document]
Comment (rogeralloymagnetic) [reply _docker/drupal/drupal-filesystem/web/config/sync/beryllium.settings.yml]: @robpblake @LightGuard I've updated the favicon and installed select2. Currently merged with master and passing tests. Waiting on stakeholder feedback.

**Comment by rogeralloymagnetic** [comment]
Closing in favor of 2817 which has Beryllium and Compose.

**Pull request closed without merging**